### ISSUE TITLE
bundletester times out and then throws a traceback

### ISSUE THREAD

**Issue by mbruzek** [juju-solutions/bundletester#56]
I was running bundletester on AWS and at the end of tests bundletester threw a python Traceback: 

```
DEBUG:root:Waiting for applications to be removed...
DEBUG:root: Remaining applications: [u'filebeat', u'elasticsearch', u'topbeat', u'kibana']
DEBUG:root: Remaining applications: [u'filebeat', u'elasticsearch', u'topbeat', u'kibana']
DEBUG:root: Remaining applications: [u'filebeat', u'elasticsearch', u'topbeat', u'kibana']
DEBUG:root: Remaining applications: [u'filebeat', u'elasticsearch', u'topbeat', u'kibana']
DEBUG:root: Remaining applications: [u'filebeat', u'elasticsearch', u'topbeat', u'kibana']
DEBUG:root: Remaining applications: [u'filebeat', u'elasticsearch', u'topbeat', u'kibana']
DEBUG:root: Remaining applications: [u'filebeat', u'elasticsearch', u'topbeat', u'kibana']
DEBUG:root: Remaining applications: [u'filebeat', u'elasticsearch', u'topbeat', u'kibana']
DEBUG:root: Remaining applications: [u'filebeat', u'elasticsearch', u'topbeat', u'kibana']
DEBUG:root: Remaining applications: [u'filebeat', u'elasticsearch', u'topbeat', u'kibana']
DEBUG:root: Remaining applications: [u'filebeat', u'elasticsearch', u'topbeat', u'kibana']
DEBUG:root: Remaining applications: [u'filebeat', u'elasticsearch', u'topbeat', u'kibana']
DEBUG:root: Remaining applications: [u'filebeat', u'elasticsearch', u'topbeat', u'kibana']
DEBUG:root: Remaining applications: [u'filebeat', u'elasticsearch', u'topbeat', u'kibana']
DEBUG:root: Remaining applications: [u'filebeat', u'elasticsearch', u'topbeat', u'kibana']
ERROR:runner:Timeout exceeded. Failed to destroy all applications  in 60 seconds.
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/bundletester/runner.py", line 179, in _run_test
    self.builder.reset()
  File "/usr/local/lib/python2.7/dist-packages/bundletester/builder.py", line 137, in reset
    ' in %s seconds.' % timeout)
RuntimeError: Timeout exceeded. Failed to destroy all applications  in 60 seconds.
DEBUG:bundletester.utils:Updating JUJU_ENV: "amazon:kube2" -> ""
Traceback (most recent call last):
  File "/usr/local/bin/bundletester", line 9, in <module>
cs:trusty/kibana-14
    load_entry_point('bundletester==0.8.0', 'console_scripts', 'bundletester')()
  File "/usr/local/lib/python2.7/dist-packages/bundletester/tester.py", line 149, in entrypoint
    status = main()
  File "/usr/local/lib/python2.7/dist-packages/bundletester/tester.py", line 141, in main
    [report.emit(result) for result in run()]
  File "/usr/local/lib/python2.7/dist-packages/bundletester/reporter.py", line 145, in emit
    cmd = message.test
  File "/usr/local/lib/python2.7/dist-packages/bundletester/reporter.py", line 14, in __getattr__
    return self[k]
KeyError: 'test'
```

It looks like it is trying to get `message.test` and that fails because there is no key `test`.

**Comment by ryan-beisner** [comment]
This should be resolved in https://github.com/juju-solutions/bundletester/pull/59, to the extent that you can now increase the reset timeout. :cat2:

**Comment by johnsca** [comment]
This was fixed in #80
